### ISSUE TITLE
Various purely decorative images should be hidden from screen readers

### ISSUE THREAD

**Issue by hs4man21** [bluesky-social/social-app#645]
**Describe the bug**
There are various purely decorative images in the web app (and possibly mobile apps as well) that are read aloud to screen readers and impede the experience of navigating Bluesky. These images should be wrapped with `aria-hidden="true"` wherever adding `alt=""` to the img tag is not enough or not possible (e.g. for svgs).

**To Reproduce**

1. Run the web app locally.
2. Using [the screen reader VoiceOver](https://build.washingtonpost.com/resources/accessibility/screen-readers#VoiceOver%20(MacOS%20and%20iOS)), navigate around the images shown below using the arrow keys and notice that - despite holding little to no meaning for navigation - they are read aloud to the screen reader. This can be annoying for users.

**Expected behavior**
Purely decorative images should be hidden from screen readers. `alt=""` will do the job for img tags but not when they are within links (as in the example below where a redundant profile link is included with the avatar images) or svgs.

For cases where `alt=""` isn't enough, we need to use `aria-hidden` on the appropriate elements or their wrappers so that screen readers gracefully ignore these empty and annoying elements.

**Screenshots**
<img width="1470" alt="Screenshot 2023-05-15 at 10 58 40 PM" src="https://github.com/bluesky-social/social-app/assets/38192823/6405a8f4-d73a-41f3-83af-7cf85cd9be16">
<img width="1150" alt="Screenshot 2023-05-15 at 10 57 54 PM" src="https://github.com/bluesky-social/social-app/assets/38192823/32eea2b1-67f3-49f6-9f97-1f7db03882de">
<img width="714" alt="Screenshot 2023-05-15 at 10 57 06 PM" src="https://github.com/bluesky-social/social-app/assets/38192823/0cd77826-322c-4bf9-999d-ee20bd5e0721">

**Details**

- Platform: Desktop, Chrome, MacOS, using VoiceOver screen reader
- Platform version: MacOS: Ventura 13.1, Chrome: 112.0.5615.137
- App version: 1.28(?), not sure where to find web app version number
